queries.sql

```diff
@@ -68,4 +68,34 @@ SELECT a.name FROM animals a
 	
 SELECT o.full_name, count(a.owner_id) FROM animals a JOIN owners o 
 	on a.owner_id = o.id group by owner_id	
+
+-----------------------------------------------------------------------------------
+SELECT a.name as Animal_Name, vi.date_of_visit from animals a 
+	JOIN visits vi On a.id = vi.animal_id JOIN vets ve On ve.id = vi.vets_id
+	WHERE ve.name = 'William Tatcher' ORDER BY vi.date_of_visit DESC Limit 1;
+
+SELECT COUNT(DISTINCT animal_id) AS Num_of_Animals FROM visits 
+	JOIN vets ON visits.vets_id = vets.id JOIN animals ON visits.animal_id = animals.id
+	WHERE vets.name = 'Stephanie Mendez';
+
+SELECT vets.name as Vets_Name, species.name AS Specialty FROM vets
+	LEFT JOIN specializations sp ON vets.id = sp.vets_id
+	LEFT JOIN species ON sp.species_id = species.id	ORDER BY vets.name;
+
+SELECT a.name, a.date_of_birth, a.neutered, a.weight_kg FROM visits
+	JOIN vets ON visits.vets_id = vets.id JOIN animals a ON visits.animal_id = a.id
+	WHERE vets.name = 'Stephanie Mendez' AND visits.date_of_visit BETWEEN '2020-04-01' AND '2020-08-30';
+	
+SELECT a.name as Animal_Name, num_visits FROM animals a
+	 JOIN (SELECT animal_id, COUNT(*) as num_visits FROM visits
+	   GROUP BY animal_id ORDER BY num_visits DESC LIMIT 1) 
+	   AS visits_count ON a.id = visits_count.animal_id;
+
+SELECT a.name AS Animal_Name, vs.date_of_visit FROM visits vi
+	JOIN animals a ON vi.animal_id = a.id JOIN vets On vets.id = vi.vets_id
+	WHERE vets.name = 'Maisy Smith' ORDER BY vs.date_of_visit ASC LIMIT 1;
+
+SELECT a.name AS Animal_Name, a.date_of_birth, a.escape_attempts, a.neutered, a.weight_kg, 
+	ve.name AS Vet_Name, ve.age as Vet_Age, ve.date_of_graduation, vi.date_of_visit FROM animals a 
+	JOIN visits vi ON vi.animal_id = a.id JOIN vets ve On ve.id = vi.vets_id ORDER BY vi.date_of_visit DESC LIMIT 1 ;
```

Comment:
- Kindly make sure to add two more queries that are missing in your `query.sql` file which are :
       - How many times visits where with a vet that did not specialize in that animal's species?
       - What specialty should Maisy Smith consider getting? Look for the species she gets the most.
       
Kindly add that too.